src/client/debugger/jupyter/debuggingManager.ts

```diff
@@ -226,7 +212,19 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDis
     private async startDebugging(doc: NotebookDocument, cell?: NotebookCell, options?: DebugSessionOptions) {
         let dbg = this.notebookToDebugger.get(doc);
         if (!dbg) {
-            dbg = new Debugger(doc, cell, options);
+            const name = cell
+                ? `${path.basename(doc.uri.toString())}?RBL=${cell.index}`
```

Comment:
Why `RBL=` Does it mean anything? Why not just `xyz.ipynb?Cell=123`
`RBL` just looks weird